### DISCUSSION TITLE
PV-Energie reicht nicht zum "betanken" des Autos...

### DISCUSSION THREAD

**Discussion by lissi514** [camueller/SmartApplianceEnabler#440] [Allgemein]
... und kann erst über den erforderliche Netzbezug im SHM erfolgen/ergänzt werden. Die Schwierigkeit besteht dabei in der mehr oder weniger permanenten manuellen Anpassung. Kann SAE diese Steuerung übernehmen?

**Comment by camueller** [comment]
Eigentlich muss man im SAE nichts anpassen - die Frage deutet eher darauf hin, wass etwas falsch konfiguriert ist oder die Funktionsweise nicht ganz verstanden worden ist.

Die Auswahl eines Ad-Hoc-Lademodus über die Ampel oder ein entsprechender Zeitplan decken alle notwendigen Anwendungsfälle ab.

**Comment by lissi514** [comment]
Ich stelle fest, dass ich unzureichend formuliert habe und mache einen neuen Anlauf. Steht ausreichend PV-Energie zur Verfügung arbeitet der SHM mit den angeschlossenen Verbrauchern einwandfrei. Die Prio der Wallbox ist untergeordnet. Steht also für die Wallbox weniger PV-Ertrag zur Verfügung als das Auto benötigt (mind. 1700 Watt), versuche ich den vorhandene PV-Ertrag zu verwenden in dem ich den Anteil an Netzstrom anpasse. Ich würde es in dieser Diskussion "manuelles Regeln" nennen, weil ich die Begriffe später nochmals verwende. Insgesamt ist das manuelle Regeln ein ziemlich nerviges unterfangen.

Nun habe ich den SAE installiert und bisher lediglich die Wallbox von SMA, EVC22-3AC-10, darin als Gerät angelegt. Meine Idee war, die weiteren Einstellungen von Ihnen kostenpflichtig vornehmen zu lassen. 

In diesem Kontext habe ich mich gefragt, ob das manuelle Regeln auch von dem SAE übernommen wird, sobald eben nicht ausreichend PV-Ertrag zur Verfügung steht.

**Comment by camueller** [comment]
Die SMA-Wallbox EVC22-3AC-10 kann durch den Smart Appliance Enaber nicht gesteuert werden. Sie kommuniziert direkt mit dem Sunny Home Manager und muss über die entsprechende App von SMA gesteuert werden.

Fragen zur SMA-Wallbox und deren Konfiguration müssen sie an SMA adressieren.

**Comment by lissi514** [comment]
Vielen Dank für Ihre Rückmeldung. Im Moment habe ich keine weiteren Verbraucher, die gesteuert werden müssten. Vielleicht kommt Ihre Lösung ja dennoch bald bei mir zum Einsatz und wünsche Ihnen viel Erfolg weiterhin!
